### PR TITLE
Some new fixes for qcbm plotter python script

### PR DESCRIPTION
**Pull request by 1ethanhansen**
I was updating the script for the docs, thought it would be good to have it updated here too 😄 

Main difference is class -> stepName and some matplotlib axes changes

Also, not sure what branch to do the PR into, so I chose dev. If you want another one, lmk and I can re-do it

### CODE REVIEW

File: examples/plot_qcbm_opt_history.py [hunk 1]
```diff
@@ -35,12 +35,12 @@ def get_ordered_list_of_bitstrings(num_qubits):
 number_of_qubits = 4
 for step_id in data:
     step = data[step_id]
-    # if step["class"] == "get-initial-parameters":
+    # if step["stepName"] == "get-initial-parameters":
     #     number_of_qubits = int(
```
Comment (martamau) [inline examples/plot_qcbm_opt_history.py:39]: This was commented as the worflowresult json in v1 didn't include the input parameters. I think one of the latest changed by Sabee added this capability so maybe its worth checking and updating?

Comment (1ethanhansen) [reply examples/plot_qcbm_opt_history.py:39]: Okay, will do!

Comment (1ethanhansen) [reply examples/plot_qcbm_opt_history.py:39]: Okay, I'll check with him on that today!

**Review by max-radin — APPROVED** [review]

File: examples/plot_qcbm_opt_history.py (hunk 1 above)
Comment (max-radin) [reply examples/plot_qcbm_opt_history.py:39]: Platform is still working on adding inputs to the workflowresult, so let's merge this PR and then we can update the logic for determining number of qubits later.

**Pull request merged**